.eslintrc.json

```diff
@@ -86,7 +86,18 @@
           "lastName",
           "profileImage",
           "tags",
-          "content"
+          "content",
```

Comment:
@MCFrank16 why are you ignoring all these things?

Reply:
esLint is complaining about them not being under the prop-types, so once they are there the eslint is resolved

Reply:
hmmm